extensions/vscode/src/api/resources/ConnectCloud.ts

```diff
@@ -0,0 +1,69 @@
+// Copyright (C) 2025 by Posit Software, PBC.
+
+import { AxiosInstance } from "axios";
+import { CONNECT_CLOUD_ENV } from "../../constants";
+import {
+  DeviceAuth,
+  AuthToken,
+  ConnectCloudAccount,
+} from "../types/connectCloud";
+
+const envHeader = { "Connect-Cloud-Environment": CONNECT_CLOUD_ENV };
+
+export class ConnectCloud {
```

Comment:
This makes me wonder if we will need nested resources for Connect Cloud at some point in the future. 🤔 Not something to do now, but planting the seed for a future discussion.